src/identity.js

```diff
@@ -746,7 +762,7 @@ export class Identity extends EventEmitter {
             max_age: maxAge,
             locale,
             one_step_login: oneStepLogin || '',
-            prompt: !acrValues ? 'select_account' : ''
+            prompt: prompt ? prompt : !acrValues ? 'select_account' : ''
```

Comment:
How about changing that logic, and set `prompt` default value to `select_account`? this line could look like `prompt: acrValues ? '' : prompt`